docker/run_crontabber.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+# Runs crontabber.
+
+# Run crontabber sleeping 5 minutes between runs
+while true; do
+    python socorro/cron/crontabber_app.py
+    echo "Sleep 5 minutes..."
+    sleep 300
+done
```

Comment:
I think we should move this conversation to this bug which is related:

https://bugzilla.mozilla.org/show_bug.cgi?id=1389571